core/pom.xml

```diff
@@ -42,6 +48,19 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>io.opentracing</groupId>
+            <artifactId>opentracing-mock</artifactId>
+            <scope>test</scope>
```

Comment:
Is there a reason that you don't specify versions of your dependencies?

Reply:
It is; see the parent pom scope under `dependencyManagement`.  If I missed one that that would be a different story.